lightly_studio/docs/docs/index.md

```diff
@@ -806,3 +806,207 @@ with open("export.txt", "w") as f:
 ### API Reference
 
 See the [Python API](api/dataset.md) for more details on the python interface.
+
+## Plugins
+
+LightlyStudio offers the possibility to extend its functionality by using plugins. Users can define their own plugins or use available ones.
+
+LightlyStudio plugins pair backend logic (the operator) with an optional frontend component. After you register an operator through the Python API, the GUI lists it automatically. For standard operators, the UI generates the input dialog for you.
+
+In the following, we will see how operators are defined, and how we can use them to define a new plugin.
+
+### Operator
+
+The core unit of a plugin is an operator. To create one you define some metadata, list the input parameters, and implement an execute method. Every operator follows the [`BaseOperator`](api/plugin/#lightly_studio.plugins.base_operator.BaseOperator) schema:
```

Comment:
I would:
- make the docs such that the operator plugin is one type of plugin
- remove the repetition of what needs to be defined.
```suggestion
An operator plugin is defined by the following attributes of the [`BaseOperator`](api/plugin/#lightly_studio.plugins.base_operator.BaseOperator) schema:
```